src/sentry/middleware/integrations/parsers/msteams.py

```diff
@@ -40,6 +40,8 @@ def get_integration_from_request(self) -> Integration | None:
         integration = self.get_integration_from_card_action(data=self.request_data)
         if integration is None:
             integration = self.get_integration_from_channel_data(data=self.request_data)
+        if integration is None:
+            integration = self.get_integration_for_tenant(data=self.request_data)
         if integration:
             return Integration.objects.filter(id=integration.id).first()
         return None
```

Comment:
At this point if the integration is still none is that a bug or is it ok? Do we want to log this?

Reply:
it's okay if it's none, this is to prevent spoofs and actual integrations that don't exist